app/src/main/res/values/strings.xml

```diff
@@ -1574,6 +1576,16 @@
     <string name="addresses_cancel_button">Cancel</string>
     <!-- The text for the "Delete address" button for deleting an address -->
     <string name="addressess_delete_address_button">Delete address</string>
+    <!-- The title for the "Delete address" confirmation dialog message-->
+    <string name="addressess_confirm_dialog_message">Are you sure you want to delete this address?</string>
+    <!-- The text for the "Delete address" button in confirmation dialog when deleting an address -->
+    <string name="addressess_confirm_dialog_ok_button">Delete</string>
+    <!-- The text for the "Cancel delete" button in confirmation dialog when deleting an address -->
```

Comment:
The comments for these strings are a little weird to me because we're saying "Delete address" button since the string for it is "Delete" in the dialog, but I would imagine you're talking about the button in the editor as well, but you also said "Cancel delete" button which is clearly not a thing. 

I would prefer if we stuck to the same language we have used in our previous positive and negative buttons for the dialogs.